src/MergeContext.js

```diff
@@ -1311,6 +1315,9 @@ class PullRequest {
             else
                 this._removePositiveStagingLabels();
 
+            if (this._prState && this._prState.brewing() && this._abandoned)
+                this._labels.add(Config.abandonedStagingChecksLabel());
```

Comment:
If `this._abandoned` is true, then we have already set the label by calling this._labels.addImmediately(). Why do we need to add it again?